great_expectations/data_context/data_context.py

```diff
@@ -1177,6 +1227,26 @@ def build_data_docs(self, site_names=None, resource_identifiers=None):
 
         return index_page_locator_infos
 
+    def clean_data_docs(self, site_name=None):
```

Comment:
Add tests to verify functionality with different stores

Reply:
Added